lib/cucumber/cli/options.rb

```diff
@@ -101,19 +101,19 @@ def parse!(args) # rubocop:disable Metrics/AbcSize
 
           opts.on("#{RETRY_FLAG} ATTEMPTS", *retry_msg) {|v| set_option :retry, v.to_i }
           opts.on('--i18n-languages', *i18n_languages_msg) { list_languages_and_exit }
-          opts.on('--i18n-keywords LANG', *i18n_keywords_msg) {|lang| set_language lang }
+          opts.on('--i18n-keywords LANG', *i18n_keywords_msg) {|lang| language lang }
```

Comment:
If these are methods with argument(s), it's actually better to call them inside parentheses. Again, not sure if you wanna fix here. There's a few examples of why this gets ambiguous with multiple args. With 1 there's no real issue, so up to you really.

Reply:
Thanks for the eye @luke-hill!

Reply:
@luke-hill There are many, many of those in that file. Thank you for calling them out though! Upon further review/thought, I'm going to leave them for now for two reasons:

1. This PR is for the Rubocop requirement around [Accessor Method Names](
http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/AccessorMethodName)
2. The style decision made in [this PR.](https://github.com/cucumber/cucumber-ruby/pull/1022)

I appreciate you calling it out though! :D

Reply:
Re-read #2 - Figure if people have made this decision elsewhere then there's no reason to alter it back. Good for digging up the refs though